src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        List<Truck> trucks = new ArrayList<>();
+        Truck truck1 = new Truck();
+        Truck truck2 = new Truck();
+        Truck truck3 = new Truck();
```

Comment:
If the `Truck` class has a constructor that requires parameters or if there are attributes that need to be initialized, you should provide those values here. Otherwise, these instances will be created with default values, which might not be intended.